recipe/find-libintl.h.patch

```diff
@@ -1,8 +0,0 @@
---- glib-2.0.pc.in.orig	2015-12-23 14:06:46.000000000 -0800
```

Comment:
Do we have any proof that we don't need this on install and use in a dependency?